### PR TITLE
Fix LSAT Logging

### PR DESCRIPTION
**Pull request by mnicpt**

### CODE REVIEW

File: src/api/api.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ export function callSmartAPI({ accessToken, url, method = 'get', headers: reqHea
                 throw new Error(`Api: ${ url } returned ack: ${ body.ack } (Corr ID: ${ headers[HEADERS.PAYPAL_DEBUG_ID] })`);
             }
 
-            return body.data;
+            return { data: body.data, headers };
```
Comment (bluepnume) [inline src/api/api.js:85]: LGTM, can we reflect the same in the type def?

File: src/api/order.js [hunk 2]
```diff
@@ -69,14 +69,22 @@ export function createOrderID(order : OrderCreateRequest, { facilitatorAccessTok
     });
 }
 
-const handleRestAPIResponse = (err, orderID : string, action : string) => {
-    getLogger().info(`call_rest_api_failure_${ action }`, { err: stringifyError(err), orderID });
+const handleRestAPIResponse = (err, orderID : string, action : string) : string => {
+    // $FlowFixMe
+    const { headers } = err.response;
```
Comment (bluepnume) [inline src/api/order.js:74]: Can we do `const { headers = {} } = err.response || {};` for safety?

File: src/api/api.js (hunk 1 above)
Comment (mnicpt) [reply src/api/api.js:90]: @bluepnume Not so sure we should do this now.  This breaks compatibility on `actions.subcription.get()`. Maybe I can not alter that signature and still provide type for `callSmartAPI`

Comment (mnicpt) [reply src/api/api.js:90]: `getSubscription` and `activateSubscription` just return the result from `callSmartAPI`

Comment (mnicpt) [reply src/api/api.js:90]: I'll use `then(` to convert to just the body data being returned.

**Comment by mnicpt** [comment]
Working on tests.

**Review by gregjopa — APPROVED** [review]

**Pull request merged**